app/lib/frontend/backend.dart

```diff
@@ -522,30 +522,35 @@ class GCloudPackageRepository extends PackageRepository {
       }
     });
 
-    final uploaderEmails =
-        await accountBackend.getEmailsOfUserIds(package.uploaders);
-
-    // Notify uploaders via e-mail that a new version has been published.
-    await emailSender.sendMessage(
-      createPackageUploadedEmail(
-        packageName: newVersion.package,
-        packageVersion: newVersion.version,
-        uploaderEmail: user.email,
-        authorizedUploaders:
-            uploaderEmails.map((email) => EmailAddress(null, email)).toList(),
-      ),
-    );
+    try {
+      final uploaderEmails =
+          await accountBackend.getEmailsOfUserIds(package.uploaders);
+
+      // Notify uploaders via e-mail that a new version has been published.
+      await emailSender.sendMessage(
+        createPackageUploadedEmail(
+          packageName: newVersion.package,
+          packageVersion: newVersion.version,
+          uploaderEmail: user.email,
+          authorizedUploaders:
+              uploaderEmails.map((email) => EmailAddress(null, email)).toList(),
+        ),
+      );
 
-    // Trigger analysis and dartdoc generation. Dependent packages can be left
-    // out here, because the dependency graph's background polling will pick up
-    // the new upload, and will trigger analysis for the dependent packages.
-    await analyzerClient
-        .triggerAnalysis(newVersion.package, newVersion.version, <String>{});
-    await dartdocClient
-        .triggerDartdoc(newVersion.package, newVersion.version, <String>{});
+      // Trigger analysis and dartdoc generation. Dependent packages can be left
+      // out here, because the dependency graph's background polling will pick up
+      // the new upload, and will trigger analysis for the dependent packages.
+      await analyzerClient
+          .triggerAnalysis(newVersion.package, newVersion.version, <String>{});
+      await dartdocClient
+          .triggerDartdoc(newVersion.package, newVersion.version, <String>{});
 
-    if (finishCallback != null) {
-      await finishCallback(newVersion);
+      if (finishCallback != null) {
+        await finishCallback(newVersion);
```

Comment:
Please document in a single line what this is used for?

Reply:
I think we should probably remove this, it is no longer used. I'm not sure why we had it in the first place, but after a while we were using it for triggering the analysis, and then after a refactor we've got a new place for that.
Should I do this in the PR?

Reply:
I filed https://github.com/dart-lang/pub-dev/issues/2304